car-style-changing.js

```diff
@@ -0,0 +1,39 @@
+var CarLot = (function carStylesModule(carLot = {}) {
+  carLot.resetBorders = () => {
+    let cars = document.getElementsByClassName('car');
+    let carObjects = carLot.getInventory();
+
+    for (let i = 0; i < cars.length; i++) {
+      cars[i].style.border = `3px solid ${getCarBorderColor(i)}`;
+      cars[i].style.backgroundColor = '#fff';
+    }
+
+    function getCarBorderColor(id) {
+      return carObjects[id].color;
+    }
+  };
+
+  // When any element inside the div with the class 'car'
+  // is clicked, traverse up the DOM until the wrapping
+  // div with the class 'car' is found, and then apply
+  // a larger border and change the background color.
+
+  carLot.selectCar = (carElement, color) => {
+    var node = carElement.parentNode;
+    while (node != null) {
```

Comment:
Most of the time this ends up being okay, but other times I get caught in an endless loop.
It only break or continues `if (node.classList)`, you don't have anything to catch an instance where there isn't a classList.
There are much easier ways to do this, but I want to give you points for ingenuity though...